src/components/Calculator.js

```diff
@@ -1,55 +1,30 @@
 import React from 'react';
+import Calculate from '../logic/calculate';
+import Display from './Display';
+import ButtonPanel from './ButtonContainer';
 
-import '../App.css';
+class Calculator extends React.Component {
+  constructor() {
```

Comment:
## Hi, You did Nice Job and your App is working fine. But you did not implement all the requirements for the purpose of this Milestone. 
- Please use `prop`. `Props` need to get passed to child components of the Calculate.js component. Please Open the Link 👉[ Props in react](https://reactjs.org/docs/components-and-props.html). to Learn more
- Also use `lifecycle` Methods. Here is the Link 👉 [lifecycle](https://reactjs.org/docs/state-and-lifecycle.html).
- You can Read this one too: [lifecycle React](https://medium.com/@amanpreet_singh/lifecycle-of-react-native-component-2020-edition-1cec4762859b#:~:text=Mounting%3A%20In%20this%20phase%2C%20component,gets%20removed%20from%20actual%20DOM.)

Reply:
Hello @OLIPLICHE 
I used state to store the current state object in Calculator.js. and this component is used in app.js with no props since it has a predefined state object. Not sure how to refactor this to use props any lead will be appreciated